### DISCUSSION TITLE
albumthumbnail.sh throws an error upon launch

### DISCUSSION THREAD

**Discussion by kreativkod** [rern/rAudio#956] [Bug Reports]
@rern  
What is happening here? Is it the album title "( )" that throws the script off, or is it the permission that acts up?  I don't understand the [31m- parts. The directory in question has 755 permissions.
![image](https://user-images.githubusercontent.com/21079205/175772984-da04a858-3ae9-442a-a1f1-f6dfed0232aa.png)

**Comment by rern** [comment]
Try it with local storage:
- Copy the directory to `/mnt/MPD/SD`
- Library | update database > Only changed files
- SD > directory - context menu > Update thumbnails
